modules/developers-guide/examples/graphql/schema-new.graphql

```diff
@@ -0,0 +1,65 @@
+type Address @cql_entity(target: UDT) @cql_input {
```

Comment:
Hmmm .... do you mean to update GraphQL docs as well? This does not seem related to Docs API "where" clauses :thinking: